content/blog/2022/11/2022-11-17-jenkins-election-candidates.adoc

```diff
@@ -0,0 +1,131 @@
+---
+:layout: post
+:title: "Jenkins Elections Announcement"
+:tags:
+- community
+- governance
+- governance-board
+- elections
+authors:
+- kmartens27
+description: >
+    Announcement for Jenkins Board elections 2022.
+links:
+  discourse: true
+opengraph:
+  image: /images/governance/elections/2022/2022-opengraph.png
+---
+
+Hello all, voter registration and candidate nominations for the 2023 Jenkins elections are now complete.
+Thanks to everyone who registered to vote and to all those who submitted nominations.
```

Comment:
```suggestion
Thanks to everyone who registered to vote and to all of those who submitted nominations.
```
(Not sure at all about this suggestion, but it sounds better to me)

Reply:
My grammar skills aren't great, but I think it is better without the additional "of".